common/main/sc.xml

```diff
@@ -1363,19 +1363,19 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 					<intervalFormats>
 						<intervalFormatFallback>{0} – {1}</intervalFormatFallback>
 						<intervalFormatItem id="Bh">
-							<greatestDifference id="B">h B – h B</greatestDifference>
+							<greatestDifference id="B">h B – h B</greatestDifference>
```

Comment:
this wouldn't have been necessary if the value in root were normalized; is there a reason that root isn't normalized?

Reply:
this is "hard inheritance" (from root) except for normalization; the lack of root normalization may make it harder for us to see what's really going in our v44 inheritance work

Reply:
The space normalization (to \u2009\u2013\u2009) is only for locales in scripts Latn, Cyrl, Grek; if we normalized root then it would have the wrong fallback for all other locales.

Reply:
(And locales in scripts Latn, Cyrl, Grek ar more likely to have vetters providing explicit values, and not rely on root fallback)